### PR TITLE
[DCS-569] Valkiria tasks

### PR DESCRIPTION
**Pull request by eruizgar**

### CODE REVIEW

File: dcos_valkiria/cli.py [hunk 1]
```diff
@@ -7,11 +7,18 @@
             [--config-file=<path>]
             [--user=<user>]
             (--ips=<ip-or-hostname>)
+    dcos valkiria tasks [--option SSHOPT=VAL ...]
```
Comment (adoblas) [inline dcos_valkiria/cli.py]: Does it make sense if in stead of "dcos valkiria tasks" we use "dcos valkiria task" like the dcos command "dcos task"?

File: dcos_valkiria/cli.py [hunk 2]
```diff
@@ -100,26 +113,82 @@ def _install(ips, user, option, config_file):
      :returns: process return code
      """
     ssh_options = util.get_ssh_options(config_file, option)
-    if not ips:
-        emitter.publish(DefaultError("--ips requires argument"))
-        return 0
-    ips = ips.split(" ")
 
-    for ip in ips:
+    # if not ips:
```
Comment (albertostratio) [inline dcos_valkiria/cli.py]: Could you please get rid of these comments?

File: dcos_valkiria/cli.py [hunk 3]
```diff
@@ -28,14 +35,15 @@
     --version
```
Comment (adoblas) [inline dcos_valkiria/cli.py]: typo?
List of ips **where** you want to install valkiria.

File: dcos_valkiria/cli.py [hunk 4]
```diff
@@ -100,26 +113,81 @@ def _install(ips, user, option, config_file):
...
+                            table.add_row([ip, 'No {} tasks running'.format(service_type)])
+                except KeyError:
+                    emitter.publish(DefaultError("{0} not contain`{1}` tasks running".format(ip, service_type)))
+        except subprocess.CalledProcessError:
+            table.add_row([ip, 'IP not valid'])
+
+    return table
+
+
+def _get_ips_list(ips):
```
Comment (albertostratio) [inline dcos_valkiria/cli.py]: This method is a good candidate to start including unit tests, isn't? Would you mind including unit tests for it?

File: tests/unit/test_cli.py [hunk 5]
```diff
@@ -1,8 +1,21 @@
...
+        self.assertEquals(cli._get_ips_list(None), 1)
+        self.assertEquals(cli._get_ips_list('10.200.1.21'), ['10.200.1.21'])
+        self.assertEquals(cli._get_ips_list('10.200.1.21,13.12.12.123'), ['10.200.1.21', '13.12.12.123'])
+        self.assertEquals(cli._get_ips_list('123.123.123.123 123.123.123.124'), ['123.123.123.123', '123.123.123.124'])
+        self.assertEquals(cli._get_ips_list('123.123.123.123 123.123.123.124 , 152.245.112.152'), ['123.123.123.123', '123.123.123.124', '152.245.112.152'])
+        self.assertEquals(cli._get_ips_list('333.200.1.21'), ['333.200.1.21'])
+        self.assertEquals(cli._get_ips_list('200.1.21'), [])
+        self.assertEquals(cli._get_ips_list('1233.200.1.21'), [])
+        self.assertEquals(cli._get_ips_list('1233.200.1.21 23.23.23.23'), ['23.23.23.23'])
+
```
Comment (albertostratio) [inline tests/unit/test_cli.py]: Nice one!! Thank you for adding it @eruizgar !!

**Pull request merged**